src/test/java/pro/trevor/tankgame/util/TestUtilities.java

```diff
@@ -0,0 +1,38 @@
+package pro.trevor.tankgame.util;
+
+import org.json.JSONObject;
+
+import pro.trevor.tankgame.rule.impl.version3.Tank;
+import pro.trevor.tankgame.rule.impl.version3.TankAttribute;
+
+public class TestUtilities 
+{
+    public static Tank buildTestTank(int actions, int gold, boolean dead)
+    {
+        JSONObject json = new JSONObject();
+        JSONObject attributes = new JSONObject();
+        attributes.put(TankAttribute.ACTIONS.name(), actions);
+        attributes.put(TankAttribute.GOLD.name(), gold);
+        attributes.put(TankAttribute.DEAD.name(), dead);
+        json.put("name", "test");
+        json.put("attributes", attributes);
+        json.put("type", "tank");
+        json.put("position", "A1");
+        return new Tank(json);
+    }
+
+    public static Tank buildDurableTestTank(int actions, int gold, int durability, boolean dead)
```

Comment:
Does Java have default arguments? I suspect this is going to be very bloated as more and more tests start requiring various combinations of some attributes but not the other attributes.

Reply:
The only way to implement something like default arguments in Java is to overload a member function of a class, i.e., 
```java
class Foo {
    void bar(int i) { ... }
    void bar()
        { 
            bar(0);
        }
}
```
This is why I suggested the idea of a builder class.

Reply:
Deferred